code/game/objects/items/weapons/chaplain_items.dm

```diff
@@ -0,0 +1,171 @@
+	set category = "Object"
+	set src in usr
+
+	if(use_check_and_message(user, USE_FORCE_SRC_IN_USER))
+		return
+
+	var/picked = input("What form would you like your obsidian relic to take?", "Reassembling your obsidian relic") as null|anything in nullchoices
+
+	if(use_check_and_message(user, USE_FORCE_SRC_IN_USER))
+		return
+	if(!ispath(nullchoices[picked]))
+		return
+
+	var/obj/item/weapon/nullrod/chosenitem = nullchoices[picked]
+	new chosenitem(get_turf(user))
+	qdel(src)
+	user.put_in_hands(chosenitem)
+
+/obj/item/weapon/nullrod/attack(mob/M as mob, mob/living/user as mob)
+
+	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN)
+	user.do_attack_animation(M)
+
+	if(LAZYLEN(user.spell_list))
+		user.silence_spells(300) //30 seconds
+		to_chat(user, span("danger", "You've been silenced!"))
+		return
+
+	if(!user.IsAdvancedToolUser())
+		to_chat(user, span("danger", "You don't have the dexterity to use this!"))
+		return
+
+	if((user.is_clumsy()) && prob(50))
+		to_chat(user, span("danger", "The [src] slips out of your hand and you hit yourself!"))
+		visible_message(span("danger", "[user] fumbles with the [src] and hits themselves in the process!"))
+		user.take_organ_damage(10)
+		user.Paralyse(20)
+		return
+
+	if(M.stat !=2 && ishuman(M) && user.a_intent != I_HURT)
+		var/mob/living/K = M
+		if(cult && (K.mind in cult.current_antagonists) && prob(75))
+			if(do_after(user, 15))
+				K.visible_message(span("danger", "[user] waves \the [src] over \the [K]'s head, [K] looks captivated by it."), span("warning", "[user] waves the [src] over your head. <b>You see a foreign light, asking you to follow it. Its presence burns and blinds.</b>"))
+				var/choice = alert(K,"Do you want to give up your goal?","Become cleansed","Resist","Give in")
+				switch(choice)
+					if("Resist")
+						K.visible_message(span("warning", "The gaze in [K]'s eyes remains determined."), span("notice", "You turn away from the light, remaining true to the Geometer!"))
+						K.say("*scream")
+						K.take_overall_damage(5, 15)
+					if("Give in")
+						K.visible_message(span("notice", "[K]'s eyes become clearer, the evil gone, but not without leaving scars."))
+						K.take_overall_damage(10, 20)
+						cult.remove_antagonist(K.mind)
+			else
+				user.visible_message(span("warning", "[user]'s concentration is broken!"), span("warning", "Your concentration is broken! You and your target need to stay uninterrupted for longer!"))
+				return
+		else
+			to_chat(user, span("danger", "The [src] appears to do nothing."))
+			M.visible_message(span("danger", "\The [user] waves \the [src] over \the [M]'s head."))
+			if(ishuman(M))
+				var/mob/living/carbon/human/H = M
+				if(prob(25))
+					H.cure_all_traumas(cure_type = CURE_SOLITUDE)
+				else if(prob(20))
+					H.cure_all_traumas(cure_type = CURE_CRYSTAL)
+			return
+		M.attack_log += text("\[[time_stamp()]\] <font color='orange'>Is being deconverted with the [src.name] by [user.name] ([user.ckey])</font>")
```

Comment:
Logs are at the wrong position.
If they are not a cultist it still logs as "attempted to deconvert"
Also log if the deconversion was a success or a failure.
Use admin_attack_log instead of adding it to the attack logs and messaging the admins individually